### ISSUE TITLE
update() does'nt update

### ISSUE THREAD

**Issue by assistans-johannes** [diegomura/react-pdf#2329]
I'm trying to use the update() function from usePDF, that's suppose to update the 

const [instance, update] = usePDF({
    document: <PDF data={data}  />,
  });

I'm expecting the update function to "re-render" my PDF document with new data that's been fetched, but i can't seem to make it work.

**Comment by jeetiss** [comment]
fixed with release #2323

**Comment by chattago2002** [comment]
Hi.
I updated to latest version to correct update() problem.

When I set the instance:

```
const [instance, update] = usePDF({document: <DocPDF d1={data1} d2={data2} d3={data3} />});
```
This is how I call update():

```
useEffect(() => {
    if (data1 && data2 && data3) {
    updateInstance()
    }
}, [data1, data2, data3]);
```

Now when I try to update the pdf instance

```
useEffect(() => {
    console.log(instance);
}, [instance]);
```

 I get the following error:

```
{
    "url": null,
    "blob": null,
    "error": {
        "message": "Cannot read properties of null (reading 'props')",
        "stack": "TypeError: Cannot read properties of null (reading 'props')\n    at _callee$ (http://localhost:3000/static/js/bundle.js:113722:40)\n    at tryCatch (http://localhost:3000/static/js/bundle.js:250800:17)\n    at Generator.<anonymous> (http://localhost:3000/static/js/bundle.js:250881:22)\n    at Generator.next (http://localhost:3000/static/js/bundle.js:250825:21)\n    at asyncGeneratorStep (http://localhost:3000/static/js/bundle.js:250660:24)\n    at _next (http://localhost:3000/static/js/bundle.js:250679:9)\n    at http://localhost:3000/static/js/bundle.js:250684:7\n    at new Promise (<anonymous>)\n    at http://localhost:3000/static/js/bundle.js:250676:12\n    at render (http://localhost:3000/static/js/bundle.js:113749:19)"
    },
    "loading": false
}
```

Where is my mistake?
